.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
```

Comment:
I think we'd probably want `exempt-all-milestones`? If we assigned a milestone, we likely do not want it closed.